src/autogluon/bench/frameworks/multimodal/exec.py

```diff
@@ -81,6 +98,21 @@ def run(
     hyperparameters: dict = None,
```

Comment:
I just noticed this while I was reading through these changes — technically I think any `kwarg` that defaults to `None` should have type `Optional`. Thoughts?

Reply:
I wanted `hyperparameters` to be provided in the args list as it is used in 
```
predictor.fit(
        train_data=train_data,
        hyperparameters=hyperparameters,
        time_limit=time_limit,
    )
```
If it is optional, then `hyperparameters` can be undefined.